### ISSUE TITLE
Trucking Assoc

### ISSUE THREAD

**Issue by rt-2** [rt-2/SA-ARP#82]
Needed for:
 - Selling trucks, trailers and forklifts;
 - Recover trucks, trailers and forklifts (sada should not be able to recover them);

Should be at blueberry

**Comment by JamesCole1999** [comment]
Do you need a place in blueberry to set that up?

**Comment by rt-2** [comment]
@JamesCole1999 Yes, was thinking of the 'truck terninal' or maybe somwhere else similar to that

**Comment by JamesCole1999** [comment]
> @JamesCole1999 Yes, was thinking of the 'truck terninal' or maybe somwhere else similar to that

There alot of place where the terminal can be, plus blueberry is the main place in game where the trucking mission happen :)

**Comment by rt-2** [comment]
I agree that blueberry should host the trucking assoc.
thre is actually a place called 'trucking terminal' in south blueberry.

**Comment by JamesCole1999** [comment]
Yeah ik the place maybe i can get a place up there if it dosent cost too much lol

**Comment by rt-2** [comment]
Some vehicles have been towed by Yanick to the Trucking Assoc, even tho it's not created

**Comment by rt-2** [comment]
Object mapping done for this. Removed fences and added doors to make an office.

**Comment by rt-2** [comment]
Sell trailer checklist:
 * [ ] must create a script that make sure no one in any circomstances cant attach to a trailer for sell (must not be too laggy like a thousand detach per sec);
 * [ ] /selltrailer to sell the currently attached trailer;
 * [ ] Script to buy;

Idea: When you attach to a trailer for sell it asks if you wanna buy, if you refuse it detaches you and also moves you forward a bit;
